### ISSUE TITLE
Waveforms in Chart Editor is not showing properly for mono sounds

### ISSUE THREAD

**Issue by Raltyro** [ShadowMario/FNF-PsychEngine#7409]
### Describe your bug here.

https://user-images.githubusercontent.com/78720179/159449189-51ba79fd-6b25-4317-a354-9c1707911100.mp4

^

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

No

**Comment by Murilo3127** [comment]
bro you activated the waveform for the instrumental

**Comment by Raltyro** [comment]
> bro you activated the waveform for the instrumental

Yes i did, and im supposed to chart the instrumental, and because the instrumental audio is mono (not stereo), it doesnt display properly

**Comment by KutikiPlayz** [comment]
make it stereo then
